src/scss/grommet-core/_objects.drop.scss

```diff
@@ -3,9 +3,19 @@
 .drop {
   position: fixed;
   z-index: 20;
-  background-color: $background-color;
+  background-color: rgba(255, 255, 255, 0.9);
```

Comment:
How about making this rgba($background-color, 0.9)? That will allow better theme customization.